applications/concept-cat/src/main/resources/application.yml

```diff
@@ -19,6 +19,7 @@ application:
   # harvest every sixth hour
   harvestCron: 0 0 */6 * * *
   harvestsourcesuris:
+    urlOverride : ${CONCEPT_CAT_HARVEST_URL_OVERRIDE:}
     uriList:
```

Comment:
Som alltid, Spring har hundre løsning i alle problemer:)

Jeg har sett i noen plass, at application configuration er allerede overskrivbar med miljøvariablene, hvis miljøvariablene skrives i spesiell måte.
kanskje det finnes litt her
https://docs.spring.io/spring-boot/docs/current/reference/html/howto-properties-and-configuration.html

En alternativløsnign  kunne være at vi bare leser listen i miljøvariablene, Internt default kunne være ingen list, og vi kan hjelpe utviklerne å sette variablene i .envrc og .envrc.template

Reply:
Jeg ønsker i utgangspunktet at så lite som mulig skal være i environment variable, slik at mest mulig konfigurasjon kommer med i filene, og under kildekodekontroll. 
Jeg så litt på linken du nevnte, uten å se noen annerledes løsning. Jeg ønsker også at det skal være mest mulig "oppdagbart" å se overrides og hva de heter.